babel/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -171,6 +171,34 @@ boolean IsNullable() :
     )
 }
 
+SqlColumnAttribute ColumnAttributeDefault() :
+{
+    SqlNode defaultValue;
+}
+{
+    <DEFAULT_>
+    (
+        defaultValue = OptionValue()
+    |
+        <NULL> {
+            defaultValue = SqlLiteral.createNull(getPos());
+        }
+    |
+        // The DateTime functions are singled out to allow for arguments to
```

Comment:
nit: Suggest moving this to a function-level comment since it applies to multiple branches in this parse clause.

Reply:
Sure, you mean moving it right above the declaration right?

Reply:
Yeah, just above "SqlColumnAttribute ColumnAttributeDefault() :"